tests/testthat/test-run_filter.R

```diff
@@ -0,0 +1,33 @@
+patrick::with_parameters_test_that(
+  "test run_filter.R",
```

Comment:
To keep it consistent with other test names.

```suggestion
  "test run_filter",
```